app/dao/notifications_dao.py

```diff
@@ -339,6 +339,7 @@ def get_notifications_for_service(
 
     if not include_jobs or (key_type and key_type != KEY_TYPE_NORMAL):
         filters.append(Notification.job_id.is_(None))
+        filters.append(Notification.api_key_id.isnot(None))
```

Comment:
We now have all four combinations of api key and job id in the system! 🎉 

If you keep `job_id.is_(None)` then this'll exclude letters from the API, since they have a job_id.
So this should just be `api_key_id != None`

|            | api key |  job id | 
|------------|---------|---------| 
| csv        | null    |         | 
| api        |         | null    | 
| letter api |         |         | 
| one off    | null    | null    |